src/PotentialEnergyEquationTerms.jl

```diff
@@ -99,11 +102,12 @@ KernelFunctionOperation at (Center, Center, Center)
     grid = model.grid
     ρ = seawater_density(model; geopotential_height)
     Z = model_geopotential_height(model)
-    parameters = (g = model.buoyancy.model.gravitational_acceleration,)
+    parameters = (g = model.buoyancy.model.gravitational_acceleration,
+                  ρ₀ = model.buoyancy.model.equation_of_state.reference_density)
```

Comment:
This line will fail unless you're using an equation of state from `SeawaterPolynomials`

Reply:
Yep that is correct but `validate_buoyancy` will throw an error if anything other than a model with `SeawaterBuoyancy{<:BoussinesqEquationOfState}` is passed. I aim to add methods for models with `SeawaterBuoyancy{<:LinearEquationOfState}` and `BuoyacnyTracer` soon.